client/components/feature-modules/blocks/components/render/list/ContentBlockList.tsx

```
@@ -34,8 +48,97 @@ interface ContentBlockListProps {
+            {isUniBlock && (
+                <div className="flex items-center justify-between">
+                    <span className="text-sm font-medium">List Mode:</span>
+                    <div className="flex items-center gap-2">
+                        <button
+                            className={`px-3 py-1 text-xs rounded transition-colors ${
+                                isManualMode
+                                    ? "bg-primary text-primary-foreground"
+                                    : "bg-muted text-muted-foreground hover:bg-muted/80"
+                            }`}
+                            onClick={() => onModeChange("MANUAL")}
+                        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)